### PR TITLE
Initial commit for AQuaMuSe

### PR DESCRIPTION
**Pull request by Karthik-Bhaskar**
There is an issue in generation of dummy data. Tests on real data have passed locally.

### CODE REVIEW

**Comment by Karthik-Bhaskar** [comment]
@yjernite Thank you for your help, generating the dummy data 🤗 Having that all the tests have passed 👍🏻

File: datasets/aquamuse/README.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
+---
+annotations_creators:
+- crowdsourced
+- expert-generated
+language_creators:
+- crowdsourced
+- expert-generated
+languages:
+- en
+licenses: []
```
Comment (lhoestq) [inline datasets/aquamuse/README.md:10]:
```suggestion
licenses:
- unknown
```

File: datasets/aquamuse/README.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+multilinguality:
+- monolingual
+size_categories:
+- 1K<n<10K
+source_datasets:
+- extended|other-Google-Natural-Questions-and-Common-Crawl
```
Comment (lhoestq) [inline datasets/aquamuse/README.md:16]:
```suggestion
- extended|natural_questions
- extended|other-Common-Crawl
```

File: datasets/aquamuse/README.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+### Citation Information
+
+    @misc{kulkarni2020aquamuse,
+          title={AQuaMuSe: Automatically Generating Datasets for Query-Based Multi-Document Summarization}, 
+          author={Sayali Kulkarni and Sheide Chammas and Wan Zhu and Fei Sha and Eugene Ie},
+          year={2020},
+          eprint={2010.12694},
+          archivePrefix={arXiv},
+          primaryClass={cs.CL}
+    }
```
Comment (lhoestq) [inline datasets/aquamuse/README.md:166]:
```suggestion
@misc{kulkarni2020aquamuse,
      title={AQuaMuSe: Automatically Generating Datasets for Query-Based Multi-Document Summarization}, 
      author={Sayali Kulkarni and Sheide Chammas and Wan Zhu and Fei Sha and Eugene Ie},
      year={2020},
      eprint={2010.12694},
      archivePrefix={arXiv},
      primaryClass={cs.CL}
}
```

File: datasets/aquamuse/README.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+- `input_urls`: a `list` of `string` features. 
+- `query`: a `string` feature.
+- `target`: a `string` feature
+
+
+Example: 
+
+    {'input_urls': ['https://boxofficebuz.com/person/19653-charles-michael-davis'],
+     'query': 'who is the actor that plays marcel on the originals',
+     'target': "In February 2013, it was announced that Davis was cast in a lead role on The CW's new show The Originals, a spinoff of The Vampire Diaries, centered on the Original Family as they move to New Orleans, where Davis' character (a vampire named Marcel) currently rules."}
```
Comment (lhoestq) [inline datasets/aquamuse/README.md:86]:
````suggestion
```
{
    'input_urls': ['https://boxofficebuz.com/person/19653-charles-michael-davis'],
     'query': 'who is the actor that plays marcel on the originals',
     'target': "In February 2013, it was announced that Davis was cast in a lead role on The CW's new show The 
Originals, a spinoff of The Vampire Diaries, centered on the Original Family as they move to New Orleans, where 
Davis' character (a vampire named Marcel) currently rules."
}
```
````

File: datasets/aquamuse/README.md [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+
+This dataset contains versions of automatically generated datasets for abstractive and extractive query-based multi-document summarization as described in [AQuaMuSe paper](https://arxiv.org/pdf/2010.12694.pdf).
+### Supported Tasks and Leaderboards
+
+- **Abstractive** and **Extractive** query-based multi-document summarization
+- Question Answering
+
+### Languages
+
+    en : English
```
Comment (lhoestq) [inline datasets/aquamuse/README.md:71]:
```suggestion
en : English
```

File: datasets/aquamuse/aquamuse.py [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+import os
+from os import listdir
+from os.path import isfile, join
+
+import tensorflow as tf
+
+import datasets
+
+
+_CITATION = """@misc{kulkarni2020aquamuse,title={AQuaMuSe: Automatically Generating Datasets for Query-Based Multi-Document Summarization}, author={Sayali Kulkarni and Sheide Chammas and Wan Zhu and Fei Sha and Eugene Ie}, year={2020}, eprint={2010.12694}, archivePrefix={arXiv}, primaryClass={cs.CL}}"""
```
Comment (lhoestq) [inline datasets/aquamuse/aquamuse.py:28]:
```suggestion
_CITATION = """\
@misc{kulkarni2020aquamuse,
      title={AQuaMuSe: Automatically Generating Datasets for Query-Based Multi-Document Summarization}, 
      author={Sayali Kulkarni and Sheide Chammas and Wan Zhu and Fei Sha and Eugene Ie},
      year={2020},
      eprint={2010.12694},
      archivePrefix={arXiv},
      primaryClass={cs.CL}
}
"""
```

File: datasets/aquamuse/aquamuse.py [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+
+        if self.config.name == "abstractive":
+            data_dir = dl_manager.download_and_extract(zipped_data_url)
+            print(data_dir)
```
Comment (lhoestq) [inline datasets/aquamuse/aquamuse.py:78]:
```suggestion
```

**Review by lhoestq — APPROVED** [review]
Awesome thank you for adding this one !

File: datasets/aquamuse/aquamuse.py [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+_CITATION = """\
+@misc{kulkarni2020aquamuse,
+      title={AQuaMuSe: Automatically Generating Datasets for Query-Based Multi-Document Summarization}, 
+      author={Sayali Kulkarni and Sheide Chammas and Wan Zhu and Fei Sha and Eugene Ie},
+      year={2020},
+      eprint={2010.12694},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL}
+}
+"""
```
Comment (lhoestq) [inline datasets/aquamuse/aquamuse.py:37]:
```suggestion
_CITATION = """\
@misc{kulkarni2020aquamuse,
      title={AQuaMuSe: Automatically Generating Datasets for Query-Based Multi-Document Summarization},
      author={Sayali Kulkarni and Sheide Chammas and Wan Zhu and Fei Sha and Eugene Ie},
      year={2020},
      eprint={2010.12694},
      archivePrefix={arXiv},
      primaryClass={cs.CL}
}
"""
```

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Comment by Karthik-Bhaskar** [comment]
Thank you :)

**Pull request merged**